src/Psalm/Internal/Analyzer/ClassLikeAnalyzer.php

```diff
@@ -765,7 +772,7 @@ protected function checkTemplateParams(
                     }
 
                     if (!$template_type->isMixed()) {
-                        $template_type_copy = clone $template_type;
+                        $template_type_copy = $template_type;
```

Comment:
is the second variable even necessary now?

Reply:
Fixed!